src/lektorium/repo/local/server.py

```diff
@@ -219,3 +223,28 @@ def __init__(
                     )
         finally:
             log.info('start ended')
+    
+    def session_address(self, session_id, container_name):
+        if self.sessions_domain is None:
+            return f'http://{container_name}:5000/'
+        return f'http://{session_id}.{self.sessions_domain}'
```

Comment:
Protocol and port could be class constants. Are also used in `lektor_labels` method